Library/Formula/cdrtools.rb

```diff
@@ -14,12 +17,29 @@ class Cdrtools < Formula
     sha256 "8f4e80ac5e512312227029504b52d39085c80b5f7d1f06d0911aaf0eaca1b6a5" => :mountain_lion
   end
 
+  devel do
+    url "https://downloads.sourceforge.net/project/cdrtools/alpha/cdrtools-3.02a02.tar.bz2"
+    mirror "https://fossies.org/linux/misc/cdrtools-3.02a02.tar.bz2"
+    sha256 "b5c33d6cfbe265806f24f365bdb885dfe35194ef716f4b6f809b4377ec159c05"
+  end
+
   depends_on "smake" => :build
 
   conflicts_with "dvdrtools",
     :because => "both dvdrtools and cdrtools install binaries by the same name"
 
   def install
+    # Speed-up the build by changing the definition of MK_FILES in each lib*/Makefile
+    # from "MK_FILES= NAME.mk NAME_p.mk $(NO_SHL)shlNAME.mk"
+    # to   "MK_FILES= NAME.mk           $(NO_SHL)shlNAME.mk"
+    # NAME can be any of libcdrdeflt libdeflt libedc libfile libfind libhfs librscg
+    #                    libscg libscgcmd libschily libsiconv mdigest paranoia
+    # This way the build will skip the *_p.mk makefiles and, consequently, also skip
+    # the creation of the profiled libs.
+    chmod "u+w", Dir["lib*/Makefile"], :verbose => true
```

Comment:
Oops, I added the `chmod u+w` because I was not sure whether `inreplace` would accept to modify read-only files... and later forgot to test the formula without it. Do you know if `inreplace` works on read-only files? (If it does I"ll remove the `chmod`, otherwise I'll just remove the `verbose` option.) Remark: 12 out of the 13 files to be modified by `inreplace` have `444` permissions.